example/requirements.txt

```diff
@@ -1,5 +1,5 @@
 ../../django-declarative-apis
-Django~=2.2.0
+Django >=2.2.0, <4
```

Comment:
We can probably make this `>=3.2` once we upgrade Django in VE ya?

Reply:
Yeah- I guess 2.2 won't be supported by Django anymore, so there's no reason for us to support it

Reply:
do we want `>=` then?

Reply:
Good point.  Let me ask the team.

Reply:
(I dropped 2.2)